de.fu_berlin.inf.dpp.whiteboard/src/de/fu_berlin/inf/dpp/whiteboard/ui/browser/BrowserAction.java

```diff
@@ -0,0 +1,35 @@
+package de.fu_berlin.inf.dpp.whiteboard.ui.browser;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+public class BrowserAction {
+
+    public class ActionTypes {
+        public final static String NEW = "NEW";
+        public final static String SET = "SET";
+        public final static String REMOVE = "REMOVE";
+    }
+
+    private String type;
+    private String elementType;
+    private String id;
+    private ArrayList<Map<String, String>> attributes;
+
+    public String getType() {
+        return type;
+    }
+
+    public String getElementType() {
+        return elementType;
+    }
+
+    public String getID() {
+        return id;
+    }
+
+    public ArrayList<Map<String, String>> getAttributes() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". ([squid:S1319](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1319))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=250&issues=AWY_InU0nzfF1D4JrUtX&open=AWY_InU0nzfF1D4JrUtX)

[ISSUE_KEY]: AWY_InU0nzfF1D4JrUtX